openmeetings-service/src/main/java/org/apache/openmeetings/service/mail/template/InvitationTemplate.html

```diff
@@ -16,20 +16,172 @@
   KIND, either express or implied.  See the License for the
   specific language governing permissions and limitations
   under the License.
-
 -->
 <!DOCTYPE html>
 <html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-9.xsd">
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500&display=swap');
+
+    @media only screen and (max-width: 620px) {
+      table.body h1 {
+        font-size: 28px !important;
+        margin-bottom: 10px !important;
+      }
+
+      table.body p,
+      table.body td,
+      table.body span,
+      table.body a {
+        font-size: 16px !important;
+      }
+
+      table.body .wrapper,
+      table.body .article {
+        padding: 10px !important;
+      }
+
+      table.body .content {
+        padding: 0 !important;
+      }
+
+      table.body .container {
+        padding: 0 !important;
+        width: 100% !important;
+      }
+
+      table.body .main {
+        border-left-width: 0 !important;
+        border-radius: 0 !important;
+        border-right-width: 0 !important;
+      }
+
+      table.body .btn table {
+        width: 100% !important;
+      }
+
+      table.body .btn a {
+        width: 100% !important;
+      }
+
+    }
+
+    @mwedia all {
+      body {
+        background-color: #f6f6f6;
+        font-family: sans-serif;
+        -webkit-font-smoothing: antialiased;
+        font-size: 14px;
+        line-height: 1.4;
+        margin: 0;
+        padding: 0;
+        -ms-text-size-adjust: 100%;
+        -webkit-text-size-adjust: 100%;
+        font-family: 'Montserrat', sans-serif;
+        font-size: 14px;
+      }
+
+    }
+  </style>
+</head>
+
 <wicket:extend>
-	<strong><span wicket:id="titleLbl"></span></strong><br />
-	<span wicket:id="userLbl"></span> <span wicket:id="user"></span><br/>
-	<span wicket:id="messageLbl"></span> <span wicket:id="message"></span><br/>
-	<div wicket:id="links">
-		<span wicket:id="comment_for_link1"></span><br/>
-		<a wicket:id="invitation_link1"><span wicket:id="clickMe"></span></a><br/>
-		<br/><br/>
-		<span wicket:id="comment_for_link2"></span><br/>
-		<span wicket:id="invitation_link2"></span>
-	</div>
+
+  <table role="presentation" border="0" cellpadding="0" cellspacing="0" class="body" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #f6f6f6; width: 100%;" width="100%" bgcolor="#f6f6f6">
```

Comment:
inline styling should be avoided, it will fail in case CSP will be enabled

Reply:
>inline styling should be avoided, it will fail in case CSP will be enabled

The simple styling (embedded in the head) is not readable by some mail clients, which causes a distortion of the design ; it's not displayed the way you want.

Reply:
Hi Mr @solomax , the mail clients ignore the css style in the head and so the design is not there anymore, we have the mail by default.
Do you have any recommendations?

Reply:
Hello @geoffreylgv,

I did quick search and was able to find this page: https://www.campaignmonitor.com/css/
According to it `<style>` tag should work

I'll do some internal testing and will report back

Reply:
Ok, and I dropped the design with `<style>` in the `<head>` tag on [gist](https://gist.github.com/geoffreylgv/d83bf33b104a5d33a904ff65e7b75ca1)

Reply:
Very much appreciated :)
Give me couple of days :)

Reply:
Hello @geoffreylgv,

I've checked the code, according to my tests only internal `<div>` was rendered (`<html>` and `<head>` were dropped)

I've fixed it (please see the branch `OPENMEETINGS-2774-emails`)

`TestEmailTemplate` can be used to test the actual HTML

Could you please take a look at this branch?

As you can see I've moved your `<style>` tag into `AbstractTemplatePage` (which is parent for all emails) so the styles can be re-used :))

Please let me know WDYT :)

Reply:
Hi Mr @solomax , I will take a look.

Reply:
Hi @solomax, can we have a build of this latest change (to deploy on vm/or any server) and test it to make sure everything is ok with the mail?

Reply:
@geoffreylgv let me check internally, then I'll try to set this up at demo server (or maybe merge then polish :)))